templates/vanilla/packages/client/src/mud/setupNetwork.ts

```diff
@@ -36,6 +80,7 @@ export async function setupNetwork() {
     onWrite: (write) => write$.next(write),
   });
 
+  /* Download the World state to have a local copy */
```

Comment:
> Sync on-chain state into RECS and keeps our client in sync. Uses the MUD indexer if available, otherwise falls back to the viem `publicClient` to make RPC calls to fetch MUD events from the chain.